### ISSUE TITLE
Use new components for creating a system/privacy declaration

### ISSUE THREAD

**Issue by allisonking** [ethyca/fides#2436]
### Is your feature request related to a specific problem?

https://github.com/ethyca/fidesplus/issues/522 adds new components to edit a system with. These are added in the plus repo, so we need to bring them in here to be reused via fidesui.

### Describe the solution you'd like

Reuse existing components and populate them in the `System information` and `Privacy delcarations` tabs made in https://github.com/ethyca/fides/issues/2435

### Describe alternatives you've considered, if any

_A description of any alternative solutions or features you've considered._

### Additional context

We'll need to move those components into fidesui first before importing them here

**Comment by allisonking** [comment]
This one might end up going in the reverse direction after some sprint reprioritization: we may build this first in admin-ui and import into datamap ui

**Comment by allisonking** [comment]
discussed with @TheAndrewJackson this morning and we have decided:

* Andrew will copy/pasta the latest form inputs (which have stacked vs inline) 
  * These are still changing a bit too much right now to put in fidesui, but eventually they should go inthere
  * These input will be used for the system describe component of the datamap UI
  * We opted not to just copy the whole SystemInformationForm because it has a ton of extra logic the datamap UI doesn't need or can't use (the extended form, custom metadata, redux logic, etc.)
* Allison will start building out the privacy declaration components with a mind for potentially having them exported

**Comment by TheAndrewJackson** [comment]
@allisonking Is this ticket ready to be closed and moved into ready for QA?

**Comment by allisonking** [comment]
Yup yup! Although at some point we'll want the ticket to move it into fidesui properly...

**Comment by TheAndrewJackson** [comment]
yes... so much copy/pasta
